components/card.css

```diff
@@ -0,0 +1,95 @@
+.cards-container {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+}
```

Comment:
`flex` by default takes direction as `row`. I think it would be better to remove that.